shoop/admin/views/dashboard.py

```diff
@@ -22,3 +23,7 @@ def get_context_data(self, **kwargs):
             blocks.extend(module.get_dashboard_blocks(request=self.request))
         context["activity"] = get_activity(request=self.request)
         return context
+
+    def get(self, request, *args, **kwargs):
+        send_telemetry(request)
+        return super(DashboardView, self).get(request, *args, **kwargs)
```

Comment:
No, it doesn't. The code says "send telemetry", which does not sound conditional in any way. Just renaming the function would make the code much more explicit.